pkg/cmd/uninstall/uninstall.go

```diff
@@ -31,7 +31,7 @@ type UninstallOptions struct {
 
 var (
 	uninstall_long = templates.LongDesc(`
-		Uninstalls the Jenkins X platform from a Kubernetes cluster`)
+		Uninstalls the Jenkins X platform from a Kubernetes cluster. This will remove all Jenkins X components, secrets, config and namespaces including any environment related namespaces`)
```

Comment:
@warrenbailey I removed the period before including as it's not gramatically correct :-p

Reply:
Definitely, good spot. That was terrible grammar :)